.github/workflows/build.yaml

```
@@ -57,3 +57,23 @@ jobs:
       node_type: "gpu-v100-latest-1"
       run_script: "ci/build_docs.sh"
       sha: ${{ inputs.sha }}
+  wheel-build:
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/wheels-pure-build.yml@cuda-120
+    with:
+      build_type: ${{ inputs.build_type || 'branch' }}
+      branch: ${{ inputs.branch }}
+      sha: ${{ inputs.sha }}
+      date: ${{ inputs.date }}
+      package-name: cudf
+      package-dir: python/cuxfilter
```

Try this, it might help. The `python/` directory contains the `pyproject.toml`. Note that I saw issues locally if I didn't have the trailing slash.
```suggestion
      package-dir: python/
```